MetaMask/metamask-extension
# fix: UX: Multichain: Fix Permissions DQA Items

## **Description**

Fixes a variety of DQA items for Permissions and Connections screens

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23989?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/23859

## **Manual testing steps**

1.  Go to all Permissions
2.  See no snaps items

## **Screenshots/Recordings**



### **Before**

Coming soon...

### **After**

Coming soon...

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
